### PR TITLE
Test pipeline 

### PR DESCRIPTION
**Pull request by JackMcPhillips1543**
This is a temporary PR to test GitHub actions

- In this PR I am adding the main.yml file that will be responsible for running the pipeline

Why was the change needed:
- We currently have no pipeline in place and need one

How was it changed:
- I added a .yml file to run the pipeline

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Step-time Biofeedback Project
+
+on:
+  push:
+    branches:  
+      - main
+      - feature/*
+  pull_request:
```
Comment (git-voo) [inline main.yml:8]: @JackMcPhillips1543 there are branch protection rules on main that may hinder your progress with testing this. I'd say just keep it for only feature branches and we can adopt it for main later

**Review by git-voo — CHANGES_REQUESTED** [review]
Hey @JackMcPhillips1543 , It seems the pipeline doesn't run because the yaml file is placed outside the GitHub directory. GitHub Actions can't find the file. We missed that😂

**Comment by JackMcPhillips1543** [comment]
> Hey @JackMcPhillips1543 , It seems the pipeline doesn't run because the yaml file is placed outside the GitHub directory. GitHub Actions can't find the file. We missed that😂

What do you mean by this? It is in the base of the repo is it not supposed to be there?

**Comment by JackMcPhillips1543** [comment]
Problem is fixed I will open a new pr when testing is done

**Pull request closed without merging**